nuc_morph_analysis/analyses/linear_regression/linear_regression.py

```diff
@@ -12,34 +10,65 @@
     RepeatedKFold,
     cross_validate,
 )
+from tqdm import tqdm
 from sklearn.pipeline import make_pipeline
 from sklearn.preprocessing import StandardScaler
-from tqdm import tqdm
 from nuc_morph_analysis.lib.preprocessing import global_dataset_filtering, filter_data
 from sklearn.model_selection import permutation_test_score
-from nuc_morph_analysis.lib.visualization.plotting_tools import get_plot_labels_for_metric
+from nuc_morph_analysis.lib.visualization.plotting_tools import (
+    get_plot_labels_for_metric,
+)
+import imageio
+from nuc_morph_analysis.analyses.linear_regression.select_features import (
+    get_feature_list,
+)
+from nuc_morph_analysis.analyses.linear_regression.utils import (
+    list_of_strings,
+    list_of_floats,
+)
 
 pd.options.mode.chained_assignment = None  # default='warn'
 
 warnings.simplefilter(action="ignore", category=FutureWarning)
 
-def main(cols, target, alpha_range, tolerance, save_path, cached_dataframe=None):
+
+def main(
+    cols,
+    target,
+    alpha_range,
+    tolerance,
+    save_path,
+    cached_dataframe=None,
+    save_movie=False,
+):
 
```

Comment:
main has no comments describing the variables or what it does. It would be good to add some comments/documentaiton

Reply:
Done